front/src/modules/pathfinding/components/Itinerary/ItineraryV2.tsx

```diff
@@ -108,16 +99,26 @@ const ItineraryV2 = ({ path }: ItineraryProps) => {
       </div>
       {displayTypeAndPath && (
         <div className="mb-2">
-          <TypeAndPath zoomToFeature={zoomToFeatureInItinerary} />
+          <TypeAndPathV2
+            zoomToFeature={zoomToFeatureInItinerary}
+            setPathProperties={setPathProperties}
+          />
         </div>
       )}
-      {origin && destination && (
+      {origin && destination && pathProperties && (
```

Comment:
I think we don't need to check pathProperties here
```suggestion
      {origin && destination && (
```

Reply:
Done